components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
I've tested that briefly in https://github.com/aleksandra-tarkowska/mapannotations/commit/7d911dff2f3fdfd069a591c448af20b63571dd17
and it work http://10.0.51.133/web/mapannotations/gene/?show=map.value-tea1